src/init.js

```diff
@@ -407,6 +411,41 @@ export const init = async (options) => {
 	}
 };
 
+export const setTagProtection = async function (options, details) {
+	const pattern = '*';
+	const { user } = options;
+
+	let octokit = new Octokit({
+		auth: user.token,
+	});
+
+	const { organization, name } = details;
+
+	if (!options.dev && organization && name) {
+		console.log(`Setting tag protection for ${organization}/${name}...`);
+
+		try {
+			// create tag protection rule repository
+			const tagProtectionResponse = await octokit.rest.repos.createTagProtection({
+				owner: organization,
+				repo: name,
+				pattern,
+			});
+
+			if (tagProtectionResponse?.status === 200 || tagProtectionResponse?.status === 201) {
```

Comment:
I think you only need to check `201` here? 
![Screen Shot 2023-02-23 at 3 52 07 PM](https://user-images.githubusercontent.com/13966707/221027521-d629ab51-5e5d-4628-bbf7-e48a5473bd10.png)
Maybe worth catching `403` and/or `404`?

Reply:
Ah, indeed.
I don't think we need to state what the reason was - just that it failed to create.